### PR TITLE
Used new population formula

### LINKED ISSUE #1134: Change the math for population changes
Instructions on implementation can be found at this [forum thread](https://forum.revolutionarygamesstudio.com/t/population-scaling/672).

### PR DESCRIPTION
**Pull request by icedjuro**
Closes #1134

### CODE REVIEW

File: project.godot [hunk 1]
```diff
@@ -170,11 +170,6 @@ toggle_FPS={
 "events": [ Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"alt":false,"shift":false,"control":false,"meta":false,"command":false,"pressed":false,"scancode":16777246,"unicode":0,"echo":false,"script":null)
  ]
 }
-g_hold_forward={
```
Comment (hhyyrylainen) [inline project.godot:173]: I'm pretty sure this and the RubySetupSystem changes are actually backwards, so these changes should be undone.

**Comment by icedjuro** [comment]
Nope

**Comment by hhyyrylainen** [comment]
I'd prefer to get some testing with the numbers before merging to avoid you having to make a second pull request right after adjusting them.

**Comment by buckly90** [comment]
I have tested the change, looks good to me!

**Pull request merged**